### ISSUE TITLE
Following script makes the python kernels missing

### ISSUE THREAD

**Issue by leolorenzoluis** [aws-samples/amazon-sagemaker-notebook-instance-lifecycle-config-samples#7]
I forked the following sample script and changed the package from scipy to plotly and for some reason when the notebook is started it doesn't show any python kernel. Only R and Sparkmagic?
```
#!/bin/bash

set -e

# OVERVIEW
# This script installs a single conda package in all SageMaker conda environments, apart from the JupyterSystemEnv which is a 
# system environment reserved for Jupyter.
# Note this may timeout if the package installations in all environments take longer than 5 mins, consider using "nohup" to run this 
# as a background process in that case.

sudo -u ec2-user -i <<'EOF'
# PARAMETERS
PACKAGE=plotly
# Note that "base" is special environment name, include it there as well.
conda install "$PACKAGE" --name base --yes
for env in /home/ec2-user/anaconda3/envs/*; do
    env_name=$(basename "$env")
    if [ $env_name = 'JupyterSystemEnv' ]; then
      continue
    fi
    nohup conda install "$PACKAGE" --name "$env_name" --yes
done

EOF
```

**Comment by mckev-amazon** [comment]
Hi @leolorenzoluis  - thanks for the report!

It seems that running many conda installs in parallel via nohup can cause issues that can make the conda environments unavailable in SageMaker.  In this case, because "nohup" is inside of the for loop, this causes many conda installs to occur at the same time.

In your script, consider moving the "nohup" prior to the "sudo" command, and use "sudo -b" so that the entire installation process happens in the background, instead of launching many simultaneous background processes at once.  I've also submitted a PR to make this less confusing for future users of the same script who want to do the same thing.

I tested this modification with your script and confirmed that I was able to "import plotly" after starting up.
